src/utils/events/getAnnotationFromSubgraph.ts

```diff
@@ -35,11 +35,13 @@ export const getAnnotationFromSubgraph = async (
        * be filtering these out, and show the most recent annotation, no matter
        * who sent it
        */
-      .filter(
-        ({ values: { agent, address } }) =>
-          agent.toLowerCase() === userAddress.toLowerCase() ||
-          address.toLowerCase() === userAddress.toLowerCase(),
-      ) || [];
-
+      .filter(({ address, values: { agent }, values }) => {
```

Comment:
We could also destructure the `address` from `values` (and rename it, so it doesn't conflict)